src/Workspaces/Core/Portable/Workspace/Host/DocumentService/Extensions.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.CodeAnalysis.Host
 {
     internal static class Extensions
     {
+        private const string RazorCSharp = "RazorCSharp";
```

Comment:
Call this the LspDiagnosticsName or something? As otherwise it's not terribly clear what this constant should be compared to.